src/edu/csus/ecs/pc2/ui/ShadowCompareRunsPane.java

```diff
@@ -242,18 +246,18 @@ private TableModel getUpdatedResultsTableModel() {
         
         //get the current judgement information from the shadow controller
         currentJudgementMap = shadowController.getJudgementComparisonInfo();
-        currentJudgementMap = shadowController.filterJudgmenentMap(currentJudgementMap);
+        filteredJudgementMap = shadowController.filterJudgmenentMap(currentJudgementMap);  //We don't want summaryPanel to count only mismatches for the summary. This prevents it.
```

Comment:
In general, we don't put comments at the end of lines since they tend to "get lost" (not seen), or cause long lines/wrapped lines).  Move the comment before the line.